llama-index-core/llama_index/core/readers/file/base.py

```diff
@@ -414,6 +419,8 @@ def load_file(
                 # about missing dependencies
                 raise ImportError(str(e))
             except Exception as e:
+                if raise_on_error:
+                    raise
```

Comment:
I thought you had to raise `e` explicitly, no?
Also, `Exception` is kinda broad. What do you think?

Reply:
I feel like given the breadth of loaders, `Exception` is fine here

I think it might be most pythonic to do `raise Exception("Encountered error while loading") from e`
https://stackoverflow.com/questions/9157210/how-do-i-raise-the-same-exception-with-a-custom-message-in-python/29442282#29442282